tools/Cargo.toml

```diff
@@ -61,6 +61,8 @@ tree-sitter = "0.20.9"
 tree-sitter-python = "0.20.2"
 tree-sitter-rust = "0.20.3"
 tree-sitter-typescript = "0.20.2"
+tree-sitter-java = "0.20.2"
+tree-sitter-kotlin = { git = "https://github.com/fwcd/tree-sitter-kotlin.git" }
```

Comment:
Sorry for the trouble, but I might have broken this by reverting https://github.com/fwcd/tree-sitter-kotlin/pull/106 upstream. The PR bumped the parser from 22 MB to 70 MB, which is too large (GitHub starts emitting warnings once files exceed 50 MB), we need to take some more time to get this down to a more reasonable size.

In the meantime please pin this to a commit (e.g. https://github.com/fwcd/tree-sitter-kotlin/commit/5b7ca7a611eac3234cd0a38b6257effa8888f89d).

Reply:
@fwcd Thanks so much for reaching out about this!  I will set the pin!

Reply:
I've pinned in https://github.com/mozsearch/mozsearch/pull/698